wordpress-mobile/WordPress-Android
# Hide a quickstart category until the next restart

Fixes #13851 

This PR adds "Hide" functionality that temporarily removes a Quickstart card from the my site screen when the user clicks on the "Hide" item in the popup menu. 

To test:
- Make sure the my site improvements flag is turned on and the app is restarted
- Make sure your site has QuickStart started
- Click on the hamburger button and the "Hide" item in the popup menu on a dynamic card 
- Notice the card disappear
- Try switching sites/rotating devices
- Notice the card is still gone
- Restart the app
- Notice the card is back


![hide_category](https://user-images.githubusercontent.com/1079756/106898111-124cab80-66f4-11eb-9930-1812144b52e8.gif)

PR submission checklist:
